backend/src/test/java/com/redhat/cloud/notifications/routers/NotificationResourceTest.java

```diff
@@ -1230,6 +1233,225 @@ void testUpdateBehaviorGroupDisplayNameBlank() {
         }
     }
 
+    /**
+     * Tests that a behavior group can be successfully appended to an event type.
+     */
+    @Test
+    void testAppendBehaviorEventType() {
+        final Bundle bundle = this.helpers.createBundle();
+        final BehaviorGroup behaviorGroup = this.helpers.createBehaviorGroup(DEFAULT_ACCOUNT_ID, DEFAULT_ORG_ID, "display-name", bundle.getId());
+
+        final Application application = this.helpers.createApplication(bundle.getId());
+        final EventType eventType = this.helpers.createEventType(application.getId(), "name", "display-name", "description");
+
+        final Header identityHeader = initRbacMock("tenant", DEFAULT_ORG_ID, "user", FULL_ACCESS);
+
+        final String url = String.format("/notifications/eventTypes/%s/behaviorGroups/%s", eventType.getId(), behaviorGroup.getId());
+
+        RestAssured.given()
+            .header(identityHeader)
+            .when()
+            .put(url)
+            .then()
+            .statusCode(204);
```

Comment:
FYI, there's a way with RestAssured to write tests that are extremely close to the tested REST endpoint:
```java
        RestAssured.given()        
            .header(identityHeader)
            .pathParam("eventTypeUuid", eventType.getId())
            .pathParam("behaviorGroupUuid", behaviorGroup.getId())
            .when()
            .put("/notifications/eventTypes/{eventTypeUuid}/behaviorGroups/{behaviorGroupUuid}")
            .then()
            .statusCode(204);
```
The PUT path could even be shared from the REST resource as a public constant, improving the maintainability of the code as checking where that constant is used would immediately show where the REST endpoint is tested.
```java
        RestAssured.given()        
            .header(identityHeader)
            .pathParam("eventTypeUuid", eventType.getId())
            .pathParam("behaviorGroupUuid", behaviorGroup.getId())
            .when()
            .put("/notifications" + THE_CONSTANT_PATH_EXPOSED_FROM_THE_RESOURCE)
            .then()
            .statusCode(204);
```
Feel free to keep the current test code though, I am not requesting any change here 😃

Reply:
Aaah, I didn't know this! Refactored! Regarding using constant paths, should we do this in a separate PR, maybe?

Reply:
It's completely minor. Let's do that later as part of another PR.